scripts/bench_python_cpp_algos.py

```diff
@@ -398,7 +407,8 @@ def load_graph(graph_path, edge_properties=None):
     parser.add_argument(
         "--graph",
         default="GAP-road",
-        choices=["GAP-road", "GAP-kron", "GAP-twitter", "GAP-web", "GAP-urand"],
+        choices=["GAP-road", "GAP-kron", "GAP-twitter", "GAP-web", "GAP-urand",
+        "rmat15"],
```

Comment:
I think this line is the formatting error you have. You will want to run `scripts/check_python_format.sh -fix ...` on this file.